src/main/java/db/Storage.java

```diff
@@ -0,0 +1,11 @@
+package db;
+
+import java.util.ArrayList;
+import java.util.List;
+import model.Bet;
+import model.User;
+
+public class Storage {
+    public static final List<User> userStorage = new ArrayList<>();
+    public static final List<Bet> bets = new ArrayList<>();
```

Comment:
```suggestion
    public static final List<User> users = new ArrayList<>();
    public static final List<Bet> bets = new ArrayList<>();
```
let's have our names consistent

Reply:
done